netty/src/main/java/io/grpc/netty/NettyServerBuilder.java

```diff
@@ -403,7 +417,7 @@ protected NettyServer buildTransportServer(
     }
 
     return new NettyServer(
-        address, channelType, bossEventLoopGroup, workerEventLoopGroup,
+        address, channelType, channelOptions, bossEventLoopGroup, workerEventLoopGroup,
```

Comment:
This needs to be a copy of the map.  If someone modifies the map later, existing servers would accidentally see the changes.

Reply:
Okay. How about coping the map in NettyServer constructor?

Reply:
This seems fine to me.